external/jquery-ui/LICENSE.txt~Widgets

```diff
@@ -41,4 +41,4 @@ CC0: http://creativecommons.org/publicdomain/zero/1.0/
 All files located in the node_modules and external directories are
 externally maintained libraries used by this software which have their
 own licenses; we recommend you read them, as their terms may differ from
-the terms above.
+the terms above.
```

Comment:
@arschmitz i dunno exactly, i added it back from the raw github because it was accidentally removed in a previous commit and didn't catch this.  probably a windows line ending thing.